README.md

```diff
@@ -0,0 +1,5 @@
+[Schema.org](http://schema.org) Flow and React support – schema-dot-org-markup
```

Comment:
This header is too long. Split "what" it is from the description.

```
# schema-dot-org-markup

> Flow and React support for [schema.org](http://schema.org/) markup
```

A list of deep links to module readmes would be helpful here, too.